deeppavlov/configs/syntax/syntax_ru_syntagrus.json

```diff
@@ -0,0 +1,121 @@
+    ],
+    "language": "ru_syntagrus",
+    "data_types": [
+      "train", "dev", "test"
+    ],
+    "read_syntax": true
+  },
+  "dataset_iterator": {
+    "class_name": "morphotagger_dataset"
+  },
+  "chainer": {
+    "in": ["x"],
+    "in_y": ["y_tags", "y_heads", "y_deps"],
+    "pipe": [
+      {
+        "in": [
+          "x"
+        ],
+        "class_name": "lazy_tokenizer",
+        "out": [
+          "x_words"
+        ]
+      },
+      {
+        "class_name": "bert_ner_preprocessor",
+        "vocab_file": "{BERT_PATH}/vocab.txt",
+        "do_lower_case": false,
+        "max_seq_length": 512,
+        "max_subword_length": 15,
+        "subword_mask_mode": "last",
+        "token_masking_prob": 0.0,
+        "in": ["x_words"],
+        "out": ["x_tokens", "x_subword_tokens", "x_subword_tok_ids", "pred_subword_mask"]
+      },
+      {
+        "class_name": "mask",
+        "in": ["x_subword_tokens"],
+        "out": ["x_subword_mask"]
+      },
+      {
+        "class_name": "bert_syntax_parser",
+        "n_deps": "1",
+        "state_size": 256,
+        "keep_prob": 0.1,
+        "bert_config_file": "{BERT_PATH}/bert_config.json",
+        "pretrained_bert": "{BERT_PATH}/bert_model.ckpt",
+        "attention_probs_keep_prob": 0.5,
+        "use_crf": false,
+        "return_probas": false,
+        "encoder_layer_ids": [6, 7, 8, 9, 10, 11],
+        "optimizer": "tf.train:AdamOptimizer",
+        "learning_rate": 1e-3,
+        "bert_learning_rate": 2e-5,
+        "min_learning_rate": 1e-7,
+        "use_birnn": true,
+        "learning_rate_drop_patience": 30,
+        "learning_rate_drop_div": 1.5,
+        "load_before_drop": true,
+        "clip_norm": null,
+        "save_path": "{WORK_PATH}/model_last_multilayer",
+        "load_path": "{WORK_PATH}/model_last_multilayer",
+        "in": ["x_subword_tok_ids", "x_subword_mask", "pred_subword_mask"],
+        "in_y": ["y_heads"],
+        "out": ["y_predicted_heads"]
+      }
+    ],
+    "out": [
+      "y_predicted_heads"
+    ]
+  },
+  "train": {
+    "epochs": 10,
+    "batch_size": 32,
+    "metrics": [
+      {
+        "name": "per_token_accuracy",
+        "inputs": [
+          "y_heads",
+          "y_predicted_heads"
+        ]
+      },
+      {
+        "name": "accuracy",
+        "inputs": [
+          "y_heads",
+          "y_predicted_heads"
+        ]
+      }
+    ],
+    "validation_patience": 10,
+    "val_every_n_epochs": 1,
+    "val_every_n_batches": 300,
+
+    "tensorboard_log_dir": "{WORK_PATH}/logs",
+    "show_examples": false,
+    "pytest_max_batches": 2,
+    "pytest_batch_size": 8,
+    "evaluation_targets": ["valid", "test"],
+    "class_name": "nn_trainer"
+  },
+  "metadata": {
+    "variables": {
+      "ROOT_PATH": "~/.deeppavlov",
+      "DOWNLOADS_PATH": "{ROOT_PATH}/downloads",
+      "MODELS_PATH": "{ROOT_PATH}/models",
+      "BERT_PATH": "/home/sorokin/data/embeddings/BERT/rubert_cased_v1",
```

Comment:
Are you sure about that?
Also, no downloads for this config

Reply:
Configs are not ready yet, I will fix them within an hour.